R/tokenfilter.R

```diff
@@ -189,8 +189,7 @@ prep.step_tokenfilter <- function(x, training, info = NULL, ...) {
       )
       n_words[[col_name]] <- length(unique(unlist(training[[col_name]])))
     }
-  } else {
-  }
+  } else {}
```

Comment:
We don't need the `else` anymore

```suggestion
  }
```